oca_dependencies.txt

```diff
@@ -0,0 +1,16 @@
+# List the OCA project dependencies, one per line
+# Add a repository url and branch if you need a forked version
+#
+# Examples
+# ========
+#
+# To depend on the standard version of sale-workflow, use:
+# sale-workflow
+#
+# To explicitely give the URL of a fork, and still use the version specified in
+# .travis.yml, use:
+# sale-workflow https://github.com/OCA/sale-workflow
+#
+# To provide both the URL and a branch, use:
+# sale-workflow https://github.com/OCA/sale-workflow branchname
+currency
```

Comment:
Talvez não precisaria desse arquivo.

Reply:
Este arquivo é para explicitar a dependência do módulo currency_update_rate que esta no repo [OCA/currency](http://github.com/OCA/currency)

Reply:
Justo. Não me atentei a última linha, só vi a parte comentada.

Reply:
That commented part isn't really needed, if take a look on other repositories

Reply:
I removed commented part!